.github/workflows/linux_builds.yml

```diff
@@ -8,8 +8,8 @@ env:
   SCONSFLAGS: verbose=yes warnings=extra werror=yes module_text_server_fb_enabled=yes strict_checks=yes
   DOTNET_NOLOGO: true
   DOTNET_CLI_TELEMETRY_OPTOUT: true
-  TSAN_OPTIONS: suppressions=misc/error_suppressions/tsan.txt
-  UBSAN_OPTIONS: suppressions=misc/error_suppressions/ubsan.txt
+  TSAN_OPTIONS: suppressions=../misc/error_suppressions/tsan.txt
+  UBSAN_OPTIONS: suppressions=../misc/error_suppressions/ubsan.txt
```

Comment:
I think we can use absolute paths as suggested by @WhalesState here:
https://github.com/godotengine/godot/pull/100380#issuecomment-2741070688

I'm quite surprised this error actually prevents from running the application, I would expect it to just skip the tsan/ubsan options if the path is invalid. Oh well.